lms/djangoapps/verify_student/models.py

```diff
@@ -721,8 +721,8 @@ def submit(self, copy_id_photo_from=None):
                 self.status = "must_retry"
```

Comment:
@awaisdar001  if response is not ok then whatever the error is, we save it in database as an attribute(error_msg) of `SoftwareSecurePhotoVerification` model. If you think we still need it anyhow then please do tell me so I will update it accordingly.